<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Игра в жизнь</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.bmp</normaloff>logo.bmp</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="GraphicsLayoutWidget" name="PlotView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStartPause">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт / Пауза</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tick">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Далее</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClean">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>380</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="infoBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="sizeOptions">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>410</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>433</y>
      <width>60</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Size NxN</string>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="PlotView2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="speedSlide">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>415</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1800</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>433</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>414</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>403</y>
      <width>23</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>_</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open .cell</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRetina">
   <property name="text">
    <string>Retina</string>
   </property>
  </action>
  <action name="actionKortx">
   <property name="text">
    <string>Kortx</string>
   </property>
  </action>
  <action name="actionKortX">
   <property name="text">
    <string>KortX</string>
   </property>
  </action>
  <action name="actionConway_s_Game_of_Life">
   <property name="text">
    <string>Conway's Game of Life</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
